src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/security/JwtAccessTokenDecoder.java

```diff
@@ -17,33 +16,38 @@
     private final ObjectMapper mapper;
 
     public JwtAccessTokenDecoder(
-            @Autowired ObjectMapper mapper
+        ObjectMapper mapper
     ) {
         this.mapper = mapper;
     }
 
     public Map<String, String> decode(
-            String accessToken
+        String accessToken
     ) {
         try {
 
             DecodedJWT jwt = JWT.decode(
-                    accessToken.replaceFirst("^Bearer\\s+", "")
+                accessToken.replaceFirst("^Bearer\\s+", "")
             );
 
             String accessTokenClaims = new String(
-                    Base64Utils.decodeFromString(jwt.getPayload())
+                Base64Utils.decodeFromString(jwt.getPayload())
             );
 
-            return mapper.readValue(
+            try {
```

Comment:
Ok I could well be wrong, but I reasoned that technically this second try block-catch wasn't required. the mapper throws when it can't parse the json into a Map<String, String> but we already know its good because JWT.decode (I think) would have thrown already for the same reason.  Also Sonar is suggesting in my IDE that nested try-block is bad

Reply:
Fair points. I will rework it to not be nested. We definitely have to catch `IOException`, else the class will not compile.